### DISCUSSION TITLE
How to get .fbx and .xodr files for the default town maps?

### DISCUSSION THREAD

**Discussion by CandyMandy28** [carla-simulator/carla#6000] [Q&A]
I'm working on a project with Carla where I just want to add leaves or snow onto the default towns, but I can't find the fbx and xodr files for the default towns (Town01, Town02,... etc.)
How would I export them? Or is there a place where I can get these files?

And then, when I import the modified files using the script, is it okay to keep the name as is?

**Comment by icyriverwolf** [comment]
CARLA_0.9.14\WindowsNoEditor\CarlaUE4\Content\Carla\Maps\OpenDrive

**Comment by NnaYelsel** [comment]
Hello @CandyMandy28 , could you finally add snow to your town ?

**Comment by Blyron** [comment]
Hello!

Our maps are composed by many fbx so is not possible to get a single fbx. What you can do is build CARLA from source and then open maps and export the used .uassets as fbx. ".xodr" files will be placed in  CARLA_ROOT\Unreal\CarlaUE4\Content\Carla\Maps\OpenDrive for towns from 1 to 10, for the largemaps there will be in C:\carla\Unreal\CarlaUnreal\Content\Carla\Maps\Town1X\OpenDrive

Useful links for this tasks:
Export uassets from UE to import them in Blender: https://www.youtube.com/watch?v=658Z_aD15kU&ab_channel=KeySmashStudios
How to build CARLA: https://carla.readthedocs.io/en/latest/build_carla/
